template

```diff
@@ -1,69 +1,50 @@
 #!/usr/bin/env bash
-#
-# Bitly = https://raw.githubusercontent.com/tallguyjenks/BusinessCard/master/business_card
-#
-# Write a nice business card on the terminal.
-# ╭───────────────────────────────────────────────────────╮
-# │                                                       │
-# │           Michael Tsouchlarakis / michaeltd           │
-# │                                                       │
-# │      Work:  tsouchlarakis@gmail.com                   │
-# │      FOSS:  Gentoo Linux avocado.                     │
-# │                                                       │
-# │   Twitter:  https://twitter.com/tsouchlarakismd       │
-# │       npm:  https://npmjs.com/~michaeltd              │
-# │    GitHub:  https://github.com/michaeltd              │
-# │  LinkedIn:  https://linkedin.com/in/michaeltd         │
-# │       Web:  https://michaeltd.netlify.com/            │
-# │                                                       │
-# │      Card:  curl -sL https://tinyurl.com/mtd-card|sh  │
-# │                                                       │
-# ╰───────────────────────────────────────────────────────╯
-# Font attributes, colors, bg colors
-reset="$(tput sgr0)"
-bold="$(tput bold)"
-dim="$(tput dim)"
+
+# bitly = https://raw.githubusercontent.com/vladdoster/curl-about-me/master/business_card
+
+######################################
+# Font attributes, colors, bg colors #
+######################################
+#bg_black="$(tput setab 0)"
+#bg_blue="$(tput setab 4)"
+#bg_cyan="$(tput setab 6)"
+#bg_default="$(tput setab 9)"
+#bg_green="$(tput setab 2)"
+#bg_magenta="$(tput setab 5)"
+#bg_red="$(tput setab 1)"
+#bg_white="$(tput setab 7)"
+#bg_yellow="$(tput setab 3)"
+#black="$(tput setaf 0)"
 #blink="$(tput blink)"
-underline="$(tput smul)"
+#default="$(tput setaf 9)"
 #end_underline="$(tput rmul)"
-#reverse="$(tput rev)"
 #hidden="$(tput invis)"
-#black="$(tput setaf 0)"
-red="$(tput setaf 1)"
-green="$(tput setaf 2)"
-yellow="$(tput setaf 3)"
+#reverse="$(tput rev)"
+#white="$(tput setaf 7)"
 blue="$(tput setaf 4)"
-magenta="$(tput setaf 5)"
+bold="$(tput bold)"
 cyan="$(tput setaf 6)"
-#white="$(tput setaf 7)"
-#default="$(tput setaf 9)"
-#bg_black="$(tput setab 0)"
-#bg_red="$(tput setab 1)"
-#bg_green="$(tput setab 2)"
-#bg_yellow="$(tput setab 3)"
-#bg_blue="$(tput setab 4)"
-#bg_magenta="$(tput setab 5)"
-#bg_cyan="$(tput setab 6)"
-#bg_white="$(tput setab 7)"
-#bg_default="$(tput setab 9)"
+dim="$(tput dim)"
+green="$(tput setaf 2)"
+magenta="$(tput setaf 5)"
+red="$(tput setaf 1)"
+reset="$(tput sgr0)"
+underline="$(tput smul)"
+yellow="$(tput setaf 3)"
 
 cat <<EOF
 
 ${cyan}╭───────────────────────────────────────────────────────╮
 ${cyan}│                                                       │
-${cyan}│${reset}           ${bold}${cyan}Bryan Jenks / TallGuyJenks${reset}                  ${cyan}│
+${cyan}│${reset}           ${bold}${cyan}Vlad Doster${reset}                  ${cyan}│
```

Comment:
or did you want to be on my template? xD

Reply:
No haha, I forgot I was just pushing to `master` and I made some changes after I made the PR. Let me change that back.